test/onnx/exporter/test_small_models_e2e.py

```diff
@@ -753,6 +758,7 @@ def forward(self, query, key, value):
         value = torch.rand(32, 8, 128, 64, dtype=torch.float16)
 
         onnx_program = self.export(Model(), (query, key, value), opset_version=23)
+        # onnxruntime is inline any op defined as a function and without any implemented kernel
```

Comment:
But does it support opset 23? We are running 1.18 in CI (because of the python version (3.9) used in ci)